### DISCUSSION TITLE
Input of blood composition data

### DISCUSSION THREAD

**Discussion by zt1947** [Open-Systems-Pharmacology/Forum#1448] [Questions & Problems]
Using PK Sim & MoBi I am trying to develop a model for a compound that binds covalently to blood proteins and to blood cells. I understand that the covalent binding reactions should be developed in MoBi. For each patient I have data such as blood albumin, hematocrit, hemoglobin, and erythrocyte count.  I understand that the observed blood albumin and hemoglobin can be manually input into PK Sim. How do I put the observed values for hematocrit & erythrocyte count?

**Comment by Yuri05** [comment]
moving the issue to the forum (@zt1947 please always ask the questions there)

**Comment by StephanSchaller** [comment]
@zt1947 , hematocrit is an existing parameter you can overwrite with your own value.
You could also create a dummy molecule for the RBCs within Plasma or RBC, depending on where your drug binds (inner / cytosolic structure or outer membrane). Initial value could be set by `(1-HTC)*VolumeWholeBlood/meanRBCsize` or sth. similar. Make sure to use the Avogadro constant to scale numbers to amounts.
